eng/common/TestResources/New-TestResources.ps1

```diff
@@ -187,7 +187,7 @@ $resourceGroupName = if ($CI) {
 
     "rg-{0}-$BaseName" -f ($serviceName -replace '[\\\/:]', '-').Substring(0, [Math]::Min($serviceName.Length, 90 - $BaseName.Length - 4)).Trim('-')
 } else {
-    "rg-$BaseName"
+    "$BaseName-testresources"
```

Comment:
FYI @heaths @danieljurek

Reply:
Prefacing "rg-" is an Azure best practice. I understand the intent, but remember this isn't just for us. That said, it doesn't hurt, but I do want to emphasize `$BaseName` isn't just - and shouldn't be assumed to be - a username. The original idea was this changed based on the service people are testing.

Reply:
Given our guidance perhaps we should change this to "$alias-rg" and append $basename if they provide something.

Reply:
You could simply default `$BaseName` to `$env:USER`, whichis far simpler:

```powershell
[Parameter()]
[ValidateNotNullOrEmpty()] # Required if not mandatory anymore
[string] $BaseName = $env:USER
```